dataactcore/utils/fileBOC.py

```diff
@@ -249,6 +249,8 @@ def sum_gtas_boc(session, period, year, agency_code):
     agency_filters = tas_agency_filter(session, agency_code, TASLookup)
     exists_query = session.query(TASLookup).filter(TASLookup.display_tas == boc_model.display_tas,
                                                    or_(*agency_filters)).exists()
+    ussgl_list = ['480100', '480110', '480200', '483100', '483200', '487100', '487200', '488100', '488200', '490100',
```

Comment:
* Duplicate `490800`
* While this is more straightforward, I worry about forgetting this if there's ever a new line added and a new dev doesn't know about this. Tested this locally.
```
import re

ussgl_list = list(set([re.findall(r'ussgl(\d+)_.*', col.name)[0] for col in fileb_model.__table__.columns if col.name.startswith('ussgl')]))
```